src/utils/ReservaController.java

```diff
@@ -0,0 +1,54 @@
+package utils;
+
+import java.sql.SQLException;
+import model.Hospede;
+import model.Reserva;
+
+public class ReservaController {
+    MySQL conn = new MySQL();
+    
+    public ReservaController() {
+        this.conn.conectaBanco();
+    }
+    
+    public void addReserva(Reserva reserva) {
+        String sql = "INSERT INTO Reservas VALUES("
+                + "null,"
+                + "'" + reserva.getQuarto()+ "',"
+                + "'" + reserva.getHospede()+ "',"
+                + "'" + reserva.getEntrada()+ "',"
+                + "'" + reserva.getSaida()+ "');";
```

Comment:
Nesse tá tudo ok, só preciso que você lembre quem for usar isso que na hora de setar a Entrada e Saída, eles tem que pegar o que tá no campo e transformar no seguinte formato: `yyyy-MM-dd hh:mm:ss` porque é assim que o banco está aceitando. Senão os caras vão ficar batendo cabeça em problema simples.